test/Java8andUp/playlist.xml

```diff
@@ -830,6 +828,78 @@
 			<subset>SE80</subset>
 		</subsets>
 	</test>
+<!--
```

Comment:
Thought we are currently limited to not support comments in the playlist files (blocked by and awaiting an infra issue to have XML::Parser installed on test machines).  Good if its not causing issues, though not sure the comments adds any information of value, as its pretty clear by the test name and the list of options that they are different.  I would likely get rid of the comments from the playlist.